datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx

```diff
@@ -71,51 +75,55 @@ export const SidebarStatsSection = () => {
-                    </InfoItem>
-                ) : null}
-            </StatsRow>
+            {hasLatestProfiles && (
+                <StatsRow>
+                    {latestProfile?.rowCount ? (
+                        <InfoItem
+                            title="Rows"
+                            onClick={() => routeToTab({ tabName: 'Queries' })}
+                            width={INFO_ITEM_WIDTH_PX}
+                        >
+                            <HeaderInfoBody>{countSeparator(latestProfile?.rowCount)}</HeaderInfoBody>
+                        </InfoItem>
+                    ) : null}
+                    {latestProfile?.columnCount ? (
+                        <InfoItem title="Columns" width={INFO_ITEM_WIDTH_PX}>
+                            <HeaderInfoBody>{latestProfile?.columnCount}</HeaderInfoBody>
+                        </InfoItem>
+                    ) : null}
+                </StatsRow>
+            )}
             {/* Usage Stats Entry */}
-            <StatsRow>
-                {usageStats?.aggregations?.totalSqlQueries ? (
-                    <InfoItem
-                        title="Monthly Queries"
-                        onClick={() => routeToTab({ tabName: 'Queries' })}
-                        width={INFO_ITEM_WIDTH_PX}
-                    >
-                        <HeaderInfoBody>{usageStats?.aggregations?.totalSqlQueries}</HeaderInfoBody>
-                    </InfoItem>
-                ) : null}
-                {(usageStats?.aggregations?.users?.length || 0) > 0 ? (
-                    <InfoItem title="Top Users" width={INFO_ITEM_WIDTH_PX}>
-                        <UsageFacepile users={usageStats?.aggregations?.users} maxNumberDisplayed={10} />
-                    </InfoItem>
-                ) : null}
-            </StatsRow>
+            {hasUsageStatsAggregations && (
+                <StatsRow>
+                    {usageStats?.aggregations?.totalSqlQueries ? (
+                        <InfoItem
+                            title="Monthly Queries"
+                            onClick={() => routeToTab({ tabName: 'Queries' })}
+                            width={INFO_ITEM_WIDTH_PX}
+                        >
+                            <HeaderInfoBody>{usageStats?.aggregations?.totalSqlQueries}</HeaderInfoBody>
+                        </InfoItem>
+                    ) : null}
+                    {(usageStats?.aggregations?.users?.length || 0) > 0 ? (
+                        <InfoItem title="Top Users" width={INFO_ITEM_WIDTH_PX}>
+                            <UsageFacepile users={usageStats?.aggregations?.users} maxNumberDisplayed={10} />
+                        </InfoItem>
+                    ) : null}
+                </StatsRow>
+            )}
             {/* Operation Entry */}
-            <StatsRow>
-                {latestOperation?.timestampMillis ? (
+            {latestOperation?.timestampMillis ? (
```

Comment:
should we call this hasLatestOperation?

Reply:
Sure, I update the code and push the changes.